common/src/main/java/com/microsoft/identity/common/internal/net/HttpRequest.java

```diff
@@ -87,19 +98,30 @@
         HTTP_METHODS.add(REQUEST_METHOD_PATCH);
     }
 
-    /**
-     * Value of read timeout in milliseconds.
-     */
-    public static int READ_TIMEOUT = 30000;
```

Comment:
@AdamBJohnsonx Why were these removed? It's breaking MSAL. Do you have the corresponding MSAL pr that goes with this common PR?